spec/models/crop_spec.rb

```diff
@@ -597,8 +597,17 @@
   end
 
   context "crop rejections" do
-    let!(:rejected_reason) { FactoryGirl.create(:crop, name: 'tomato', approval_status: 'rejected', reason_for_rejection: 'not edible') }
-    let!(:rejected_other) { FactoryGirl.create(:crop, name: 'tomato', approval_status: 'rejected', reason_for_rejection: 'other', rejection_notes: 'blah blah blah') }
+    let!(:rejected_reason) do
+      FactoryGirl.create(:crop, name: 'tomato',
```

Comment:
Again, I'd prefer an indentation style that didn't introduce vast acres of horizontal whitespace, but I see what you were going for here. Interesting.